public/configs/v1/env.json

```diff
@@ -85,7 +85,6 @@
          "isolatedMarginLearnMore": "https://help.dydx.trade/en/articles/172975-isolated-margin",
          "mintscanValidatorsLearnMore": "https://www.mintscan.io/dydx/validators",
          "protocolStaking": "https://protocolstaking.info/",
-         "validatorSelectionDocument": ""
```

Comment:
I think the trailing commas need to be removed as well

Reply:
oops ty my bad